spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -85,8 +85,11 @@ class Recurrent[T : ClassTag]()
     }
     var t = cells.length
     if (t < times) {
+      val cloneCell = cells.head.cloneModule()
+      cloneCell.parameters()._1.foreach(x => null)
+      cloneCell.parameters()._2.foreach(x => null)
```

Comment:
why not use Tensor.set() here? should not set tensor to null as it will loss the storage share information.